lib/auth0Bundler.js

```diff
@@ -12,6 +12,7 @@ const buildLiteralAst = require('./buildLiteralAst');
 const dependencies = { rollup, babelTransform, BabelPluginExportToFunction, buildLiteralAst, rollupCommonjs };
 
 module.exports = {
-    bundleScript: R.partial(bundleRule, [ dependencies ]),
-    bundleRule: R.partial(bundleRule, [ dependencies ])
+    bundleScript: R.partial(bundleRule, [ dependencies, { commonjs: false } ]),
```

Comment:
Maybe we should rename `bundleRule` as it now bundles everything: Just `bundle` would be ok imho.